### PR TITLE
basic log of command output for feedback

### PR DESCRIPTION
**Pull request by pr8kerl**
Dump output returned from executed shells, to allow for some feedback.

### CODE REVIEW

File: pkg/watcher/controller.go [hunk 1]
```diff
@@ -43,7 +45,19 @@ func (s *ShellReconciler) Reconcile(req reconcile.Request) (reconcile.Result, er
...
+
+	if len(output) > 0 {
+		reader := bytes.NewReader(output)
+		scanner := bufio.NewScanner(reader)
+		for scanner.Scan() {
+			glog.V(4).Infof("%s output: %s", s.ObjectKind, scanner.Text())
+		}
+
+		if err := scanner.Err(); err != nil {
+			glog.V(4).Infoln(err)
```
Comment (pvdvreede) [inline pkg/watcher/controller.go]: Should this be logged as an error or warning?

File: pkg/watcher/controller.go [hunk 2]
```diff
@@ -30,9 +32,9 @@ type ShellReconciler struct {
 }
 
 func (s *ShellReconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
-	glog.V(4).Infof("Received reconcile request for %s/%s.", req.Namespace, req.Name)
+	glog.V(4).Infof("Received reconcile request for %s", req)
```
Comment (pvdvreede) [inline pkg/watcher/controller.go]: Nice if the request object takes care of this.

File: pkg/watcher/controller.go [hunk 3]
```diff
@@ -43,7 +45,19 @@ func (s *ShellReconciler) Reconcile(req reconcile.Request) (reconcile.Result, er
 	})
 
 	glog.V(4).Infof("Running command %s...", s.Command)
-	_, err := cmd.CombinedOutput()
+	output, err := cmd.CombinedOutput()
+
+	if len(output) > 0 {
```
Comment (pvdvreede) [inline pkg/watcher/controller.go]: If the purpose is for debugging, would it be worth adding a log message to say the shell process had no output?

File: pkg/watcher/controller.go (hunk 2 above)
Comment (pr8kerl) [reply pkg/watcher/controller.go]: yep - you get `namespace/name` returned.

File: pkg/watcher/controller.go (hunk 3 above)
Comment (pr8kerl) [reply pkg/watcher/controller.go]: good idea.

File: pkg/watcher/controller.go (hunk 1 above)
Comment (pr8kerl) [reply pkg/watcher/controller.go]: I thought so too - but the V() only has Info* functions.

Comment (pvdvreede) [reply pkg/watcher/controller.go]: Hmmmm, how about we use the non V functions for error and warning as you almost always want that logged, and just use V funcs for info/debugging? So you can change this line to a .Error call.

**Pull request merged**